packages/core/components/FileMetadataSearchBar/test/FileMetadataSearchBar.test.tsx

```diff
@@ -116,26 +113,30 @@ describe("<FileMetadataSearchBar />", () => {
         expect(getByText("Sun Aug 30 2020")).to.not.be.empty;
     });
 
-    it("defaults end date to start date when only start date is chosen", async () => {
+    it("creates RANGE() file filter of RANGE(day,day+1) when only start date is selected", async () => {
         // Arrange
         const { actions, logicMiddleware, store } = configureMockStore({ state: initialState });
         const { getByText } = render(
             <Provider store={store}>
                 <FileMetadataSearchBar />
             </Provider>
         );
-        const day = 17;
-        const date = new Date();
-        const expectedDate = `${date.getFullYear()}-${("0" + (date.getMonth() + 1)).slice(
-            -2
-        )}-${day}`;
-        const expectedRange = `${expectedDate}${DATE_RANGE_SEPARATOR}${expectedDate}`;
+        const startDate = new Date();
+        const day = 15; // Arbitrary day of the month that isn't likely to render twice
```

Comment:
I don't understand what is meant by 'render twice' here.

Reply:
Notice how in this picture of the calendar component, some dates are listed twice, e.g. "30" and "1". This can screw with the testing-library function
![image](https://user-images.githubusercontent.com/65628854/188202838-49a509dc-3bfd-4f51-9995-f66d2cbeade4.png)

Reply:
maybe "Show up twice in the calender popup"?